utopia-remix/app/components/sharingDialog.tsx

```diff
@@ -127,6 +127,7 @@ function AccessRequests({
         <Flex key={request.token} justify='between'>
           <Text size='1'>{request.User?.name ?? request.User?.email ?? request.user_id}</Text>
           {status === AccessRequestStatus.PENDING ? (
+            // eslint-disable-next-line react/jsx-no-bind
```

Comment:
since it's inside a map function, the only way to avoid this errors here and in :192,  is to extract it to a different component. I'm not sure it's not an overkill here